test/CheckoutSdkTest/Extensions/CheckoutServiceCollectionTest.cs

```diff
@@ -16,6 +16,9 @@ public class CheckoutServiceCollectionTest
         private void ShouldCreateCheckoutPreviousApiSingleton()
         {
             var loggerFactoryMock = new Mock<ILoggerFactory>();
+            var mockLogger = new Mock<ILogger>();
```

Comment:
These test changes should be able to be reverted if you make the suggested change, as discussed on Slack

Reply:
👍🏼